### ISSUE TITLE
AttributeError: 'Session' object has no attribute 'new_transaction'

### ISSUE THREAD

**Issue by Nemesisesq** [neo4j/neo4j-python-driver#117]
There appears to be a discrepancy between the documentation [here](http://neo4j.com/docs/api/python-driver/current/) and the current library. there doesn't appear to be a new_transaction method on the Session object. 

```
/Users/Nem/.virtualenvs/streamsavvy_data/bin/python /Applications/PyCharm.app/Contents/helpers/pydev/pydevd.py --multiproc --qt-support --client 127.0.0.1 --port 58540 --file /Applications/PyCharm.app/Contents/helpers/pycharm/behave_runner.py
Testing started at 1:29 AM ...
pydev debugger: process 52159 is connecting

Connected to pydev debugger (build 163.9735.8)

Traceback (most recent call last):
  File "/Users/Nem/.virtualenvs/streamsavvy_data/lib/python3.5/site-packages/behave/model.py", line 1456, in run
    match.run(runner.context)
  File "/Users/Nem/.virtualenvs/streamsavvy_data/lib/python3.5/site-packages/behave/model.py", line 1903, in run
    self.func(context, *args, **kwargs)
  File "features/steps/pg_neo.py", line 14, in step_impl
    context.result = convert_content_pg_mongoneo()
  File "/Users/Nem/streamsavvy_dataprocessing/scripts/pg_neo.py", line 41, in convert_content_pg_mongoneo
    with session.new_transaction() as tx:
AttributeError: 'Session' object has no attribute 'new_transaction'

Captured logging:
INFO:neo4j.bolt:~~ [CONNECT] localhost 7687
INFO:neo4j.bolt:~~ [SECURE] localhost
INFO:neo4j.bolt:C: [HANDSHAKE] 0x6060B017 [1, 0, 0, 0]
INFO:neo4j.bolt:S: [HANDSHAKE] 1
INFO:neo4j.bolt:C: INIT 'neo4j-python/1.0.2' {'scheme': 'basic', 'principal': '***', 'credentials': '******'}
INFO:neo4j.bolt:S: SUCCESS {'server': 'Neo4j/3.1.0'}

```

**Comment by technige** [comment]
Thank you, yes this is a bug in the docs.

**Comment by technige** [comment]
Fixed in 5cb50d496a2ab3599091ce35add0909913fd1463
